src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -86,10 +90,23 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
   const [txFeeCryptoPrecision, setTxFeeCryptoPrecision] = useState<string | undefined>()
 
   const { currentAccountIdByChainId, positionStatus } = confirmedQuote
+  const { type } = fromQuote(confirmedQuote)
 
   const asset = useAppSelector(state => selectAssetById(state, assetId))
+
+  const isSymAssetWithdraw =
```

Comment:
specifically this broke my brain trying to keep track of what type was what and what this means

Reply:
what was the edge case for clarification? `assetId` should be thorchain for sym withdrawals

Reply:
Unfortunately the `assetId` is not thorchain if the `intentType` as you wish to name it is asymetrical withdrawal of `asset` on `sym` opportunityType, so we have to force the fact that we wan't to send a rune tx if the intent is to withdraw asymetricaly on the asset side for a symetrical opportunity

Reply:
- `asym` opportunityType + `asym` withdraw = assetId
- `sym` opportunityType + `asym` asset side withdraw: we should use the asset side asset id to display the correct asset but send the TX as a rune tx using rune as fee assets
If I add another case to use rune for that type of withdrawal, the whole system under would display the rune asset instead of the actual asset, we wan't to display AVAX + RUNE as fees and send a rune tx

- `sym` opportunityType + `sym` side withdraw: we wan't to display both assets and use rune TX, it works like a charm without any special cases

Example: symmetric opportunityType - asym asset withdrawal => will be sent through a RUNE tx using rune as fee asset

![image](https://github.com/user-attachments/assets/ccd70349-bf4c-4bbb-83ce-0530e5e7e040)

If we pass the thorchain asset in the `TransactionRow` component, it would display the rune asset instead of the currently selected side asset and then it would be wrong

TDLR: we can't send the `thorchainAssetId` to the `TransactionRow` component if the wanted withdrawal is: `I want to withdraw asym asset side for my sym position` because we need to display the asset but as a rune tx
It's not like we were doing an asym withdraw of an asym opportunity which would be fine because we would be sending the TX using the chain of the selected asset and the fee asset would be the asset itself

Reply:
As discussed, I was mistaken in my assumption of being able to withdraw asym asset from a sym position using an asset tx. Thanks for the clarification!